examples/storm-jms-examples/src/main/java/org/apache/storm/jms/example/ExampleJmsTopology.java

```diff
@@ -41,7 +45,12 @@
     public static final String JMS_TOPIC_SPOUT = "JMS_TOPIC_SPOUT";
     public static final String ANOTHER_BOLT = "ANOTHER_BOLT";
 
-    @SuppressWarnings("serial")
+    /**
+     * The main method.
+     * @param args takes the topology name as first argument
+     * @throws Exception any expection occuring durch cluster setup or operation
+     */
+    @SuppressWarnings({"serial", "checkstyle:VariableDeclarationUsageDistance"})
```

Comment:
This seems like a bad way to get violation count to 0. We're just hiding the violation now.

Reply:
In my opinion it's valid to make exceptions if they're thought through. Especially is the exception is associated with concrete line of code rather than just represented by a count.

Breaking up this method would reduce it's readability in my opinion. After all, I don't care, but I'm starting the discussion in order to figure out whether you're strictly against it (I might do similar work) or whether you accept it if you agree.

Reply:
I agree in principle, but in this case the violation is just saying to move the `jmsTopicProvider` declaration closer to where it is first used. I don't think you need to break up the method.